core/src/main/scala/cats/instances/all.scala

```diff
@@ -42,6 +42,7 @@ trait AllInstancesBinCompat1
     with StreamInstancesBinCompat0
     with MapInstancesBinCompat0
     with SortedMapInstancesBinCompat0
+    with SortedSetInstancesBinCompat0
```

Comment:
This is the problem right here, you can't add anything to existing traits, only final classes and objects :)

Reply:
Ah, Do I just need to add `AllInstancesBinCompat4` that extends `SortedSetInstancesBinCompat0` ?

Reply:
Yup,I believe so